tests/js/jest.config.js

```diff
@@ -13,6 +13,8 @@ module.exports = {
 		'<rootDir>/build',
 		'<rootDir>/tests/shared',
 		'<rootDir>/vendor',
+		'<rootDir>/assets/src/block-editor/plugins',
+		'<rootDir>/assets/src/block-validation/plugins',
```

Comment:
Since Gutenberg plugins do not render markup directly, covering them with unit tests is not practical. Instead, whenever needed, they should be covered by E2E tests.

By ignoring the plugins we ensure that the code coverage report is not negatively impacted.